analyzers/src/SonarAnalyzer.ShimLayer.CodeGeneration/SonarAnalyzer.ShimLayer.CodeGeneration.csproj

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.2.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.4.0" />
```

Comment:
This is one version below the latest release 4.5.0, because 4.5.0 requires VS 17.5.0 and (silently) doesn't work without that.
We need to wait some time before bumping it.